.travis.yml

```diff
@@ -36,4 +36,5 @@ script:
  - ./.buildifier.sh
  - python -m unittest discover -s reconciletags -p 'reconciletags_test.py'
  - bazel clean && bazel build //...
+ - sudo rm -rf /etc/docker
```

Comment:
is there a specific file you're trying to remove by doing this? do we need to remove the whole folder?